### PR TITLE
add api accuracy for BN slice and squeeze OP

### PR DESCRIPTION
**Pull request by wangchaochaohu**
Adding OP accuracy with tensorflow

### CODE REVIEW

File: api/common/utils.py [hunk 1]
```diff
@@ -18,37 +18,35 @@
...
  
     assert len(list1) == len(list2)
     num_outputs = len(list1)
     for i in xrange(num_outputs):
         output1 = list1[i]
         output2 = list2[i]
         try:
-            diff = compare(output1, output2)
-            max_diff = diff if diff > max_diff else max_diff
+            compare(output1, output2, max_diff)
```
Comment (Xreki) [inline api/common/utils.py]: 这样，似乎前面的output的max_diff被直接覆盖？

File: api/accuracy/batch_norm.py [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+
+    pd_feed, tf_feed = feed_random_data(pd_obj, tf_obj)
+
+    # Run Paddle
+    pd_outputs = pd_obj.run_with_executor(use_gpu=use_gpu, feed=pd_feed, check_output=False)
+
+    # Run Tensorflow
+    tf_outputs = tf_obj.run(use_gpu=use_gpu, feed=tf_feed, check_output=False)
+
+    utils.check_outputs(pd_outputs, tf_outputs, name=name)
```
[the remark this answers is not in the document]
Comment (wangchaochaohu) [reply api/accuracy/batch_norm.py]: Done

File: api/accuracy/batch_norm.py [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+                name='data', shape=[10, 10, 100, 100], dtype='float32', lod_level=0)
+            param_attr = fluid.ParamAttr(name='batch_norm_w', initializer=fluid.initializer.Constant(value=1.0))
+            bias_attr = fluid.ParamAttr(name='batch_norm_b', initializer=fluid.initializer.Constant(value=0.0))
+            data.stop_gradient = False
+            result = fluid.layers.batch_norm(input=data, param_attr = param_attr, bias_attr = bias_attr, epsilon=0.001)
+
+            self.feed_vars = [data]
+            self.fetch_vars = [result]
+            if backward:
+                self.append_gradients(result, [data])
```
[the remark this answers is not in the document]
Comment (wangchaochaohu) [reply api/accuracy/batch_norm.py]: 这样反向会出错

File: api/common/utils.py (hunk 1 above)
Comment (wangchaochaohu) [reply api/common/utils.py]: 因为原来的实现在assert之后没有赋值diff  导致就算存在diff 也看不到diff 是多大

File: api/accuracy/batch_norm.py (hunk 3 above)
[the remark this answers is not in the document]
Comment (zhaoyuchen2018) [reply api/accuracy/batch_norm.py]: 是paddle还是tf的反向会出错？

[the remark this answers is not in the document]
Comment (wangchaochaohu) [reply api/accuracy/batch_norm.py]: paddle

File: api/accuracy/batch_norm.py [hunk 4]
```diff
@@ -0,0 +1,111 @@
...
+from common import utils
+      
+class PaddleBatchNorm(paddle_api.PaddleAPIBenchmarkBase):
+    def build_program(self, backward=False):
+        self.name = "batch_norm"
+        with fluid.program_guard(self.main_program, self.startup_program):
+            data = fluid.data(
+                name='data', shape=[10, 10, 100, 100], dtype='float32', lod_level=0)
+            param_attr = fluid.ParamAttr(name='batch_norm_w', initializer=fluid.initializer.Constant(value=1.0))
+            bias_attr = fluid.ParamAttr(name='batch_norm_b', initializer=fluid.initializer.Constant(value=0.0))
```
[the remark this answers is not in the document]
Comment (wangchaochaohu) [reply api/accuracy/batch_norm.py]: 这个不好feed 和conv的filter不一样
